README.md

```diff
@@ -1,13 +1,19 @@
 # Qiskit
+
+[![Release](https://img.shields.io/github/v/release/Qiskit/qiskit-terra.svg?style=flat&filter=0.*&logo=Qiskit)](https://github.com/Qiskit/qiskit/releases?q=tag%3A0)
 [![License](https://img.shields.io/github/license/Qiskit/qiskit-terra.svg?)](https://opensource.org/licenses/Apache-2.0) <!--- long-description-skip-begin -->
-[![Release](https://img.shields.io/github/release/Qiskit/qiskit-terra.svg)](https://github.com/Qiskit/qiskit-terra/releases)
 [![Downloads](https://img.shields.io/pypi/dm/qiskit-terra.svg)](https://pypi.org/project/qiskit-terra/)
 [![Coverage Status](https://coveralls.io/repos/github/Qiskit/qiskit-terra/badge.svg?branch=main)](https://coveralls.io/github/Qiskit/qiskit-terra?branch=main)
-![PyPI - Python Version](https://img.shields.io/pypi/pyversions/qiskit)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/qiskit-terra)
 [![Minimum rustc 1.64.0](https://img.shields.io/badge/rustc-1.64.0+-blue.svg)](https://rust-lang.github.io/rfcs/2495-min-rust-version.html)
 [![Downloads](https://static.pepy.tech/badge/qiskit-terra)](https://pepy.tech/project/qiskit-terra)<!--- long-description-skip-end -->
```

Comment:
These particular badges don't appear in the PyPI package for older releases (before we moved the package configuration to be static in `pyproject.toml`), so they won't be showing on the PyPI page for `qiskit-terra`.  We used to have dynamic logic that stripped everything between `<!-- long-description-skip-begin -->` and its corresponding "end" when attaching the metadata to the Python distribution.

That said, it's probably fine to change them for the legacy stable branch anyway.